packages/phrases/src/locales/en.ts

```diff
@@ -4,6 +4,7 @@ const translation = {
     add_another: '+ Add Another',
     skip: 'Skip',
     next: 'Next',
+    try_again: 'Try again',
```

Comment:
also use `retry` as key?

Reply:
most of our i18nKeys are the same as the english text?

Reply:
If it's as simple as a single word, yes. Other than that, no...
Key should be as brief as possible, while the value should be more descriptive and more user friendly.

Reply:
Again, it's nit-picking thing, tho. Should not block anything

Reply:
Alright, I'm just curious, don't have much exp in i18n things. Updated.